corehq/ex-submodules/couchforms/migrations/0006_unfinishedarchivestub_attempts.py

```diff
@@ -13,7 +13,7 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AddField(
-            model_name='unfinishedarchivestub',
+            model_name='unfinishedarchivestu',
```

Comment:
@dannyroberts was this intentional to leave the `b` off then end in this migration? Seems like a find and replace error...